utils.py

```diff
@@ -52,7 +52,7 @@ def load_image(file1,alpha=False,subdir=''): # eg subdir='glow'
     try:
         img=pygame.image.load(fname)
     except:
-        print "Peter says: Can't find "+fname; exit()
+        print("Peter says: Can't find "+fname); exit()

```

Comment:
For consistency, you can move `exit()` to a new line.

Reply:
The semicolon at the end of the line throws me off as python doesn't require semicolons at the end of lines (or so I think). So can anyone please explain why the semicolon is there? Thank you.

Reply:
Python can use semicolons to have many lines in one single lines (one liner codes). But for the sake of PEP8 formatting, it is better not have semicolons;

```
print("Peter says: Can't find "+fname); exit()
```
is better written as 
```
print("Peter says: Can't find "+fname)
exit()
```

Reply:
Got it. Thanks :)